subsys/spm/Kconfig

```diff
@@ -44,6 +46,7 @@ menuconfig IS_SPM
 	bool "Current app is SPM"
 	default n
 	select TRUSTED_EXECUTION_SECURE
+	imply ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS
```

Comment:
This probably fits most of the use-cases, but there might be some SPMs that don't need Secure calls, and this would probably have two consequences,
- possible build errors, if you don't specify any secure calls,
- possible flash usage increase

So I wonder if this should just be removed and set explicitly in the project configuration...

Reply:
This is meant to be temporary, making sure it works in the current configuration. In the future, I imagine ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS will be set when enabling the actual secure services.

Reply:
ok